lib/inspec/resources/selinux.rb

```diff
@@ -0,0 +1,57 @@
+require "inspec/resources/command"
+
+module Inspec::Resources
+  class Selinux < Inspec.resource(1)
+    name "selinux"
+
+    supports platform: "unix"
+
+    desc "Use selinux Inspec resource to test state/mode of the selinux policy."
+
+    example <<~EXAMPLE
+      describe selinux do
+        it { should be_installed }
+        it { should be_disabled }
+        it { should be_permissive }
+        it { should be_enforcing }
+      end
+    EXAMPLE
+
+    def initialize(selinux_path = "/etc/selinux/config")
+      @path = selinux_path
+      cmd = inspec.command("sestatus")
+
+      if cmd.exit_status != 0
+        return skip_resource "#{cmd.stdout}"
```

Comment:
If there is an error, you will probably want stderr, not stdout.

Reply:
Right. Does skipping resource in case of error is fine ? .

Reply:
Yes, that's common.

Reply:
Since command not found is coming in stdout, now I am handling both.